mmpose/datasets/datasets/mesh/mesh_h36m_dataset.py

```diff
@@ -0,0 +1,109 @@
+import os
+from collections import OrderedDict
+
+import json_tricks as json
+import numpy as np
+
+from mmpose.datasets.builder import DATASETS
+from ....core.evaluation import compute_similarity_transform
+from .mesh_base_dataset import MeshBaseDataset
+
+
+@DATASETS.register_module()
+class MeshH36MDataset(MeshBaseDataset):
+    """Human3.6M Dataset for 3D human mesh estimation. It inherits all function
+    from MeshBaseDataset and has its own evaluate fuction.
+
+    The dataset loads raw features and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    Args:
+        ann_file (str): Path to the annotation file.
+        img_prefix (str): Path to a directory where images are held.
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def evaluate(self, outputs, res_folder, metric='joint_error', **kwargs):
```

Comment:
where is this `kwargs` used?

Reply:
I just follow the style of top_down_dataset and did not remove this variable. I will remove it.

Reply:
this will cause trouble when the user mistype metric as medic.
because no warning/error will be raised.

Reply:
OK, I will remove the kwargs.